README.md

```diff
@@ -83,6 +83,16 @@ php composer.phar install
 ```
 Debian/Ubuntu people, just execute `install_ubuntu.sh`
 
+## Nightly Builds
+
+Instead of setting everything up manually, you can just [download the nighly builds](https://en0ch.se/index.php/s/3v5J3XXeQCCp73p) instead. These builds are updated every 12 hour - 00:00 and 12:00, and is pre-configured with all the needed dependencies.
```

Comment:
* every 12 hour*s*
* I think it should be 'and _are_ pre-configured' instead of 'and _is_ pre-configured'